code/statsd/src/main/scala/com/avast/datadog4s/statsd/metric/DistributionDoubleImpl.scala

```diff
@@ -11,9 +11,10 @@ class DistributionDoubleImpl[F[_]: Sync](
   sampleRate: Double,
   defaultTags: Seq[Tag]
 ) extends Distribution[F, Double] {
-  private[this] val F                                     = Sync[F]
+  private val F                                           = Sync[F]
```

Comment:
This looks weird. What if you try to insert a new line after `Sync[F]`? That way it might not try to align equals operators.

Reply:
(also please fix that everywhere, not just here)